bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "origin-web-common",
-  "version": "0.0.44",
+  "version": "0.0.45",
```

Comment:
Do you mind also removing the version bump from this PR per @jeff-phillips-18's comments in the catalog PR?